src/Components/Layout.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+import Navigation from './Navigation';
+
+function Layout() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.